### ISSUE TITLE
Error: Brace indexing is not supported for variables of this type.

### ISSUE THREAD

**Issue by AnthonyZJiang** [kupiqu/plot2svg#19]
This error is thrown at [L2311](https://github.com/kupiqu/fig2svg/blob/master/src/fig2svg.m#L2311) if `markerfacecolorname` has a value of `'none'` in the previous iteration. Therefore, `markerfacecolorname` needs to be reset to an empty `cell` at the beginning or end of the for loop.

**Comment by kupiqu** [comment]
I think I encountered this issue before and fixed it, but forgot to push the changes until yesterday, more or less in parallel with your issue report...

Please try current master version and let me know. If the issue persists, please paste a brief code  example, so I can reproduce the issue.

**Comment by kupiqu** [comment]
@tmxkn1 could you please confirm if the issue is still present in master version? thanks

**Comment by AnthonyZJiang** [comment]
[**@kupiqu**](https://github.com/kupiqu) commented on [Mar 26, 2019, 1:45 PM GMT](https://github.com/kupiqu/fig2svg/issues/19#issuecomment-476644549 "2019-03-26T13:45:21Z - Replied by Github Reply Comments"):
> [@tmxkn1](https://github.com/tmxkn1) could you please confirm if the issue is still present in master version? thanks

Sorry for late reply. Yes, the problem is still there.

From my command window:

```matlab
>> fig2svg('a')
   FIG to SVG converter version 2019.03.0, maintained by Salva Ardid (https://github.com/kupiqu/fig2svg).
Unable to perform assignment because brace indexing is not supported for variables of this type.

Error in fig2svg>axchild2svg (line 2308)
                  markeredgecolorname{pointc} = searchcolor(id,c_tmp(pointc,:));

Error in fig2svg>axes2svg (line 1705)
  boundingBoxAxes = axchild2svg(fid,id,axIdString,ax,paperpos,axchild,axpos,groupax,projection,boundingBoxAxes);

Error in fig2svg (line 221)
          group = axes2svg(fid,id,ax(j),group,paperpos);
```

My MATLAB version:

```matlab
>> version

ans =

    '9.5.0.1033004 (R2018b) Update 2'
```

The figure (*.fig) is attached.

My temporary solution is inserting`markeredgecolorname = {};` before [L2068](https://github.com/kupiqu/fig2svg/blob/master/src/fig2svg.m#L2068).

There are also some other issues with plotting this figure; for example, sizes and colours of circles representing the scatter points do not match with those plotted in MATLAB. But these are different issues, and I will start a new issue post for them later on.

[31243669-img-sec-2.zip](https://github.com/kupiqu/fig2svg/files/3011060/31243669-img-sec-2.zip)

**Comment by kupiqu** [comment]
> My temporary solution is inserting `markeredgecolorname = {};` before L2068.

I reproduce the issue and can fix it with your workaround at line 2306, is that what you meant?

> There are also some other issues with plotting this figure; for example, sizes and colours of circles representing the scatter points do not match with those plotted in MATLAB.

Please open the issue and give me a tiny example of a scatter plot to reproduce, so I understand it better than just with a figure.

Thank you

**Comment by kupiqu** [comment]
I applied the same trick to markerfacecolorname
